s/elements/context/bindings/default_bindings_schema.ts

```diff
@@ -0,0 +1,32 @@
+
+import {BindingsSchema} from "./types/bindings-schema.js"
+
+export const default_mode = "humanoid"
+
+export const default_bindings_schema = {
+	[default_mode]: {
+		pointer: {
+			look: {causes: ["Pointer", "Lookpad"]},
```

Comment:
in addition to `causes`, we can now optionally include `label` and `icon` into the schema.

i haven't added the icons yet, i will do that as a part of my upcoming work on the gridmenu.